docs/patterns/components/TwoColumnPanel/index.js

```diff
@@ -0,0 +1,361 @@
+import React, { useRef, useState } from 'react';
+import {
+  TwoColumnPanel,
+  ColumnLeft,
+  ColumnRight,
+  ColumnHeader,
+  ColumnGroupHeader,
+  ColumnList,
+  Breadcrumb,
+  BreadcrumbLink,
+  BreadcrumbItem,
+  Code
+} from '@deque/cauldron-react/';
+import PropDocs from '../../../Demo/PropDocs';
+import { children, className } from '../../../props';
+
+function TwoColumnPanelDemo() {
+  const [selected, setSelected] = useState(1);
+  const [selectedGroup, setSelectedGroupItem] = useState(1);
+
+  const items = [
+    {
+      id: 1,
+      name: 'One',
+      description: 'Short description of one',
+      contents: 'Content for One'
+    },
+    {
+      id: 2,
+      name: 'Two',
+      description: 'Short description of two',
+      contents: 'Content for Two'
+    },
+    {
+      id: 3,
+      name: 'Three',
+      description: 'Short description of three',
+      contents: 'Content for Three'
+    }
+  ];
+
+  const groupOne = [
+    {
+      id: 1,
+      name: 'One',
+      description: 'Short description of one',
+      contents: 'Content for grouped One'
+    },
+    {
+      id: 2,
+      name: 'Two',
+      description: 'Short description of two',
+      contents: 'Content for grouped Two'
+    }
+  ];
+  const groupTwo = [
+    {
+      id: 3,
+      name: 'Three',
+      description: 'Short description of three',
+      contents: 'Content for grouped Three'
+    },
+    {
+      id: 4,
+      name: 'Four',
+      description: 'Short description of four',
+      contents: 'Content for grouped Four'
+    }
+  ];
+
+  const selectedItem = items.find(item => item.id === selected);
+  const selectedGroupItem = [...groupOne, ...groupTwo].find(
+    item => item.id === selectedGroup
+  );
+  const selectItem = id => event => {
+    event.preventDefault();
+    setSelected(id);
+  };
+  const selectGroupItem = id => event => {
+    event.preventDefault();
+    setSelectedGroupItem(id);
+  };
+
+  return (
+    <div className="twocolumnpanel-demo">
+      <h1>Two Column Panel</h1>
+
+      <p>
+        Two column panel is a container component intended to provide a
+        navigable list of items that can be collapsed if not needed.
+      </p>
+
+      <h2>Basic</h2>
+
+      <TwoColumnPanel>
+        <ColumnLeft aria-labelledby="sidebar">
+          <ColumnHeader id="sidebar">Items</ColumnHeader>
+          <nav aria-label="Sidebar navigation">
```

Comment:
For my own learning on accessibility, should these follow the keyboard navigation patterns for menus? https://www.w3.org/TR/wai-aria-practices/#menu

However, I noticed that the walnut app does not allow for keyboard navigation with arrow keys. Why is that?

Reply:
From a UX perspective, I wouldn't really consider this a "menu" since it's not a fixed set of items or actions but rather navigation links that change context on the page. There's also a [good article by Heydon Pickering](https://inclusive-components.design/menus-menu-buttons/) on this behavior. I think for similar reasons that are mentioned in the above article are likely why walnut does not use this pattern for its navigation.